### PR TITLE
Validate File Type in URL

### LINKED ISSUE #8: URL Validation
Users shouldn't be able to add a photo unless the URL ends with `.png`, `.jpg` etc.

### PR DESCRIPTION
**Pull request by Bouncey**
closes #8

### CODE REVIEW

File: static/js/main.js [hunk 1]
```diff
@@ -37,13 +37,24 @@ photosRef.once("value")
...
-  firebase.database().ref('photos/' + $('#title').val()).set({
-    title: $('#title').val(),
-    url: $('#url').val(),
-    location: $('input[name="location"]:checked').val(),
-    tags: $('#tags').val(),
-    likes: 0
-  });
+   var url = $('#url').val();
+
+          if(url.match(/(jpg|png)$/)){
```
Comment (ghost) [inline static/js/main.js]: @Bouncey you should allow `gif` here also. Please update the alert message as well.

**Comment by atjonathan** [comment]
#### **LGTM :+1:**
Thanks so much for spending time on this :smile:

**Comment by Bouncey** [comment]
Checks for `http` at beginning of URL and a valid file type at the end.

Works with `https`

**Comment by atjonathan** [comment]
:heart:

**Pull request merged**